crates/gitbutler-tauri/src/menu.rs

```diff
@@ -44,6 +60,9 @@ fn disabled_menu_item(id: &str, title: &str) -> MenuEntry {
 }
 
 pub fn handle_event<R: Runtime>(event: &WindowMenuEvent<R>) {
+    if event.menu_item_id() == "view/devtools" {
```

Comment:
This doesn't match the string above. Think it needs to be `app/dev-tools` here, or alternatively change the above to match this one.